### DISCUSSION TITLE
windows 11+oneapi can't buld .lib files

### DISCUSSION THREAD

**Discussion by jingui37** [scivision/mumps-superbuild#69] [Q&A] [answered]
Dear scivision:
I have received the following information after i input "cmake -G Ninja -B build -DBUILD_SINGLE=yes -DBUILD_DOUBLE=yes -DBUILD_COMPLEX=yes -DBUILD_COMPLEX16=yes"

C:\Program Files (x86)\Intel\oneAPI\mumps-main>cmake -G Ninja -B build -DBUILD_SINGLE=yes -DBUILD_DOUBLE=yes -DBUILD_COMPLEX=yes -DBUILD_COMPLEX16=yes
-- The C compiler identification is IntelLLVM 2023.2.0 with MSVC-like command-line
-- The Fortran compiler identification is Intel 2021.10.0.20230609
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files (x86)/Intel/oneAPI/compiler/latest/windows/bin/icx.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting Fortran compiler ABI info
-- Detecting Fortran compiler ABI info - done
-- Check for working Fortran compiler: C:/Program Files (x86)/Intel/oneAPI/compiler/2023.2.0/windows/bin/intel64/ifort.exe - skipped
-- MUMPS 5.6.2.2 upstream 5.6.2 install prefix: C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/local
-- CMake 3.28.0-rc2  Toolchain
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - not found
-- Found Threads: TRUE
-- Found MPI_C: C:/Program Files (x86)/Intel/oneAPI/mpi/latest/lib/release/impi.lib (found version "3.1")
-- Found MPI_Fortran: C:/Program Files (x86)/Intel/oneAPI/mpi/latest/lib/release/impi.lib (found version "3.1")
-- Found MPI: TRUE (found version "3.1") found components: C Fortran
-- MKL_ARCH: None, set to ` intel64` by default
-- MKL_LINK: None, set to ` dynamic` by default
-- MKL_INTERFACE_FULL: intel_lp64
-- MKL_THREADING: None, set to ` intel_thread` by default
-- MKL_MPI: None, set to ` intelmpi` by default
-- Found MKL: intelmpi
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64/mkl_scalapack_lp64_dll.lib
-- Found DLL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/redist/intel64/mkl_scalapack_lp64.2.dll
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64/mkl_intel_lp64_dll.lib
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64/mkl_intel_thread_dll.lib
-- Found DLL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/redist/intel64/mkl_intel_thread.2.dll
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64/mkl_core_dll.lib
-- Found DLL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/redist/intel64/mkl_core.2.dll
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/lib/intel64/mkl_blacs_lp64_dll.lib
-- Found DLL: C:/Program Files (x86)/Intel/oneAPI/mkl/latest/redist/intel64/mkl_blacs_lp64.2.dll
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/tbb/latest/../../compiler/latest/windows/redist/intel64_win/compiler
-- Found MKL: C:/Program Files (x86)/Intel/oneAPI/compiler/latest/windows/compiler/lib/intel64/libiomp5md.lib
-- Performing Test SCALAPACK_d_FOUND
-- Performing Test SCALAPACK_d_FOUND - Success
-- Performing Test SCALAPACK_s_FOUND
-- Performing Test SCALAPACK_s_FOUND - Success
-- Found SCALAPACK: $<IF:$<BOOL:OFF>,,>;MKL::mkl_scalapack_lp64;MKL::mkl_intel_lp64;MKL::mkl_intel_thread;MKL::mkl_core;MKL::mkl_blacs_lp64;C:/Program Files (x86)/Intel/oneAPI/compiler/latest/windows/compiler/lib/intel64/libiomp5md.lib  found components: MKL
-- MUMPS ORDERING_DEFS: pord
-- MUMPS ORDERING_LIBS: pord
-- Populating mumps
-- Configuring done (0.2s)
-- Generating done (0.0s)
-- Build files have been written to: C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-subbuild
[1/9] Creating directories for 'mumps-populate'
[1/9] Performing download step (download, verify and extract) for 'mumps-populate'
-- Downloading...
   dst='C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-subbuild/mumps-populate-prefix/src/MUMPS_5.6.2.tar.gz'
   timeout='none'
   inactivity timeout='60 seconds'
-- Using src='https://mumps-solver.org/MUMPS_5.6.2.tar.gz'
-- [download 0% complete]
-- [download 1% complete]
-- [download 2% complete]
-- [download 3% complete]
-- [download 4% complete]
-- [download 5% complete]
-- [download 6% complete]
-- [download 7% complete]
-- [download 8% complete]
-- [download 9% complete]
-- [download 10% complete]
-- [download 11% complete]
-- [download 12% complete]
-- [download 13% complete]
-- [download 14% complete]
-- [download 15% complete]
-- [download 16% complete]
-- [download 17% complete]
-- [download 18% complete]
-- [download 19% complete]
-- [download 20% complete]
-- [download 21% complete]
-- [download 22% complete]
-- [download 23% complete]
-- [download 24% complete]
-- [download 25% complete]
-- [download 26% complete]
-- [download 27% complete]
-- [download 28% complete]
-- [download 29% complete]
-- [download 30% complete]
-- [download 31% complete]
-- [download 32% complete]
-- [download 33% complete]
-- [download 34% complete]
-- [download 35% complete]
-- [download 36% complete]
-- [download 37% complete]
-- [download 38% complete]
-- [download 39% complete]
-- [download 40% complete]
-- [download 41% complete]
-- [download 42% complete]
-- [download 43% complete]
-- [download 44% complete]
-- [download 45% complete]
-- [download 46% complete]
-- [download 47% complete]
-- [download 48% complete]
-- [download 49% complete]
-- [download 50% complete]
-- [download 51% complete]
-- [download 52% complete]
-- [download 53% complete]
-- [download 54% complete]
-- [download 55% complete]
-- [download 56% complete]
-- [download 57% complete]
-- [download 58% complete]
-- [download 59% complete]
-- [download 60% complete]
-- [download 61% complete]
-- [download 62% complete]
-- [download 63% complete]
-- [download 64% complete]
-- [download 65% complete]
-- [download 66% complete]
-- [download 67% complete]
-- [download 68% complete]
-- [download 69% complete]
-- [download 70% complete]
-- [download 71% complete]
-- [download 72% complete]
-- [download 73% complete]
-- [download 74% complete]
-- [download 75% complete]
-- [download 76% complete]
-- [download 77% complete]
-- [download 78% complete]
-- [download 79% complete]
-- [download 80% complete]
-- [download 81% complete]
-- [download 82% complete]
-- [download 83% complete]
-- [download 84% complete]
-- [download 85% complete]
-- [download 86% complete]
-- [download 87% complete]
-- [download 88% complete]
-- [download 89% complete]
-- [download 90% complete]
-- [download 91% complete]
-- [download 92% complete]
-- [download 93% complete]
-- [download 94% complete]
-- [download 95% complete]
-- [download 96% complete]
-- [download 97% complete]
-- [download 98% complete]
-- [download 99% complete]
-- [download 100% complete]
-- verifying file...
       file='C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-subbuild/mumps-populate-prefix/src/MUMPS_5.6.2.tar.gz'
-- Downloading... done
-- extracting...
     src='C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-subbuild/mumps-populate-prefix/src/MUMPS_5.6.2.tar.gz'
     dst='C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-src'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
[2/9] No update_disconnected step for 'mumps-populate'
[3/9] No patch_disconnected step for 'mumps-populate'
[5/9] No configure step for 'mumps-populate'
[6/9] No build step for 'mumps-populate'
[7/9] No install step for 'mumps-populate'
[8/9] No test step for 'mumps-populate'
[9/9] Completed 'mumps-populate'
-- Configured MUMPS 5.6.2 source in C:/Program Files (x86)/Intel/oneAPI/mumps-main/build/_deps/mumps-src
-- The following features have been enabled:

 * Parallel, parallel MUMPS (using MPI and Scalapack)
 * real32, Build with single precision
 * real64, Build with double precision
 * complex32, Build with complex precision
 * complex64, Build with complex16 precision

-- The following features have been disabled:

 * 64-bit-integer, use 64-bit integers in C and Fortran
 * GEMMT, use GEMMT for symmetric matrix-matrix multiplication
 * METIS, METIS graph partitioning https://github.com/KarypisLab/METIS
 * parMETIS, parMETIS parallel graph partitioning
 * Scotch, Scotch graph partitioning https://www.labri.fr/perso/pelegrin/scotch/
 * Openmp, OpenMP API https://www.openmp.org/
 * shared, Build shared libraries

-- Configuring done (848.2s)
-- Generating done (2.2s)
-- Build files have been written to: C:/Program Files (x86)/Intel/oneAPI/mumps-main/build

But I can't find any .lib files in the build file. Can you please help me solve this problem? Thank you very much.

**Comment by scivision** [comment, marked as answer]
now you need to type

```sh
cmake --build build
```

that is the path to your " C:/Program Files (x86)/Intel/oneAPI/mumps-main/build"
